src/Core.UnitTests/CFamily/EffectiveRulesConfigCalculatorTests.cs

```diff
@@ -32,20 +34,23 @@ public class EffectiveRulesConfigCalculatorTests
         [TestMethod]
         [DataRow(true /* custom settings are null */)]
         [DataRow(false /* custom settings are not null, but emtpy */)]
-        public void GetConfig_NullCustomSettings_DefaultConfigReturned(bool sourceCustomSettingsAreNull)
+        public void GetConfig_NoCustomSettings_NoError_DefaultsReturned(bool customSettingsAreNull)
         {
             // Arrange
             var testLogger = new TestLogger();
```

Comment:
Please extract the repetitions to TestInitialize

Reply:
I've moved the two common lines of setup code into `TestInitialize`, and the test that relate to the inner cache class to a different file `EffectiveRulesConfigCalculatorCacheTests`.

On balance, though, I still agree with [Mr Newkirk's reasons](https://jamesnewkirk.typepad.com/posts/2007/09/why-you-should-.html) for avoiding `TestInitialize`.